flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -1784,7 +1784,7 @@ object TaskManager {
    * @return
    */
   private def getMethodToFetchCPULoad(): Option[Method] = {
-    val methodsList = classOf[com.sun.management.OperatingSystemMXBean].getMethods()
+    val methodsList = classOf[java.lang.management.OperatingSystemMXBean].getMethods()
```

Comment:
getMethods() will never contain the `getProcessCpuLoad` method, so the feature is basically disabled